src/components/buttons/Button.vue

```diff
@@ -52,19 +62,38 @@ export default defineComponent({
   appearance: none;
   font-size: 13px;
   border-radius: var(--radius-1);
-  min-width: 138px;
-  min-height: 35px;
   justify-content: center;
-  margin-top: 5px;
 
-  @media (min-width: 430px) {
-    margin-top: 0;
```

Comment:
Moving margin-top out of the styles here makes the button more reusable. The margin now comes from WidgetMainSmall